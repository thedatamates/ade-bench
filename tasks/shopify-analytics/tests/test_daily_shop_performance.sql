-- Test daily shop performance for the most recent 7 days
SELECT 
    date,
    total_orders,
    total_order_revenue,
    abandoned_checkouts_count,
    abandoned_checkouts_value,
    fulfilled_orders,
    unfulfilled_orders,
    partially_fulfilled_orders,
    avg_order_value
FROM daily_shop_performance
ORDER BY date DESC
LIMIT 7;