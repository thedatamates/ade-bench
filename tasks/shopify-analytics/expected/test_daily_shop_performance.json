[
  {
    "date": "2024-01-07",
    "total_orders": 150,
    "total_order_revenue": 15000.00,
    "abandoned_checkouts_count": 20,
    "abandoned_checkouts_value": 2000.00,
    "fulfilled_orders": 140,
    "unfulfilled_orders": 5,
    "partially_fulfilled_orders": 5,
    "avg_order_value": 100.00
  }
]